src/bin/travis/resources/jes_centaur.conf.ctmpl

```diff
@@ -96,5 +96,9 @@ database {
   profile = "slick.jdbc.MySQLProfile$"
 }
 
+backend.providers.Jes.config.concurrent-job-limit = 20
```

Comment:
what's up w/ all these config changes?

Reply:
Since all the Centaur CWLs are now eligible to run concurrently they were overwhelming Cromwell on Travis at the default job limit (10 million I think).